test/functional/fundrawtransaction.py

```diff
@@ -449,12 +449,12 @@ def run_test(self):
         ############################################################
         # locked wallet test
         self.nodes[1].encryptwallet("test")
+        self.stop_node(0)
+        self.stop_node(2)
+        self.stop_node(3)
         self.nodes.pop(1)
-        stop_node(self.nodes[0], 0)
-        stop_node(self.nodes[1], 2)
-        stop_node(self.nodes[2], 3)
 
-        self.nodes = start_nodes(self.num_nodes, self.options.tmpdir)
+        self.nodes = self.start_nodes(self.num_nodes, self.options.tmpdir)
```

Comment:
certainly something we can do in a future PR.

BitcoinTestFramework could include two functions - `start_nodes()` to start the nodes in `self.nodes`, and `add_nodes()` if you need to append nodes to that list.